checkov/sca_package_2/output.py

```diff
@@ -120,38 +122,41 @@ def create_cli_output(fixable: bool = True, *cve_records: list[Record]) -> str:
                     severity_str = record.severity.name.lower() if record.severity else BcSeverities.NONE.lower()
                     setattr(cve_count, severity_str, getattr(cve_count, severity_str) + 1)
 
+                    package_name = record.vulnerability_details["package_name"]
+                    package_version = record.vulnerability_details["package_version"]
+
                     if record.vulnerability_details["lowest_fixed_version"] != UNFIXABLE_VERSION:
                         cve_count.has_fix += 1
+                    if root_package_alias == get_package_alias(package_name, package_version):
+                        fix_versions_lists.append(record.vulnerability_details["fixed_versions"])
 
-                    fix_versions_lists.append(record.vulnerability_details["fixed_versions"])
-                    if package_version is None:
-                        package_version = record.vulnerability_details["package_version"]
-
-                    package_cves_details_map[package_name].setdefault("cves", []).append(
+                    package_cves_details_map[root_package_alias].setdefault("cves", []).append(
                         {
                             "id": record.vulnerability_details["id"],
                             "severity": severity_str,
                             "fixed_version": record.vulnerability_details["lowest_fixed_version"],
+                            "root_package_name": record.vulnerability_details["root_package_name"],
+                            "root_package_version": record.vulnerability_details["root_package_version"],
+                            "package_name": package_name,
+                            "package_version": package_version,
                         }
                     )
                 elif record.check_name == SCA_LICENSE_CHECK_NAME:
                     if record.check_result["result"] == CheckResult.SKIPPED:
                         continue
                     should_print_licenses_table = True
-                    package_licenses_details_map[package_name].append(
-                        _LicenseStatus(package_name=package_name,
+                    package_licenses_details_map[record.vulnerability_details["package_name"]].append(
+                        _LicenseStatus(package_name=record.vulnerability_details["package_name"],
```

Comment:
notice that record.vulnerability_details["package_name"]==package_name so:
```suggestion
                    package_licenses_details_map[package_name].append(
                        _LicenseStatus(package_name=package_name,
```